OpenSSL/test/test_ssl.py

```diff
@@ -2745,6 +2745,20 @@ def test_get_cipher_bits(self):
         self.assertEqual(server_cipher_bits, client_cipher_bits)
 
 
+    def test_get_protocol_version(self):
+        """
+        :py:obj:`Connection.get_protocol_version()` returns a string
+        giving the protocol version of the current connection.
+        """
+        server, client = self._loopback()
+        server_protocol_version, client_protocol_version = \
```

Comment:
Please split this into two proper lines.  The current version takes the same amount of lines and is less readable.